go/libkb/kbcmf_enc.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
+
+import (
+	"io"
+
+	"github.com/keybase/client/go/kbcmf"
+)
+
+// KBCMFEncrypt reads from the given source, encrypts it for the given
+// receivers from the given sender, armors it, and writes it to sink.
+func KBCMFEncrypt(
+	source io.Reader, sink io.WriteCloser,
+	receivers [][]NaclDHKeyPublic, sender NaclDHKeyPair) error {
+	var receiverBoxKeys [][]kbcmf.BoxPublicKey
+	for _, receiverPublicKeys := range receivers {
+		var t []kbcmf.BoxPublicKey
+		for _, k := range receiverPublicKeys {
+			t = append(t, naclBoxPublicKey(k))
+		}
+		receiverBoxKeys = append(receiverBoxKeys, t)
+	}
+	plainsink, err := newKeybaseEncryptArmor62Stream(
```

Comment:
This check is in my current PR, see [here](https://github.com/keybase/client/pull/1363/files#diff-0a7a02b982750f69354bf034729a69a3R100)